src/main/java/uk/gov/hmcts/reform/preapi/media/MediaKind.java

```diff
@@ -36,23 +49,32 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static uk.gov.hmcts.reform.preapi.media.MediaResourcesHelper.getSanitisedLiveEventId;
+import static uk.gov.hmcts.reform.preapi.media.MediaResourcesHelper.getShortenedLiveEventId;
+
+
 @Service
 public class MediaKind implements IMediaService {
     private final String ingestStorageAccount;
     private final String environmentTag;
+    private final String subscription;
 
     private final MediaKindClient mediaKindClient;
 
+    private static final Logger logger = LoggerFactory.getLogger(MediaKind.class);
```

Comment:
Should be able to use `@Slf4j` lombok annotation on class which will create a logger without this line of code (logger variable will be called `log` instead)